libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -237,16 +237,14 @@ void CodeTransform::operator()(FunctionCall const& _call)
 {
 	yulAssert(m_scope, "");
 
+	m_assembly.setSourceLocation(extractSourceLocationFromDebugData(_call.debugData));
```

Comment:
This is the actual fix.

Reply:
Curious - I actually thought that ``builtin->generateCode`` did set the source location, but apparently it doesn't.

Reply:
Or actually... doing it in ``builtin->generateCode`` before visiting the arguments as well would also fix it. Generally I'd like to change the ``generateCode`` thing and remove the callback in it and instead have the caller make sure that the arguments are on stack. But anyways, the fix is good for now.

Reply:
actually you are right, the instruction itself might have a different source location. let me fix it in some more places :)